promotion/service.go

```diff
@@ -38,3 +42,23 @@ func InitService(datastore Datastore) (*Service, error) {
 		reputationClient: reputationClient,
 	}, nil
 }
+
+// CheckJobs starts check for unfinished jobs on a ticker
+func (service *Service) CheckJobs(ctx context.Context, shouldLoop bool) {
+	ticker := time.NewTicker(1000 * time.Millisecond)
+	for {
+		attempted, err := service.datastore.RunNextClaimJob(ctx, service)
+		if err != nil {
+			raven.CaptureErrorAndWait(err, nil)
+			break
+		}
+		if !attempted {
+			raven.CaptureMessageAndWait("unable to attempt process claim job", nil)
```

Comment:
This isn't an error condition - it just means all signing jobs are being handled. I'm confused why we are sending this to sentry